src/js/editor/editor.js

```diff
@@ -384,6 +386,13 @@ class Editor {
     return activeSections[activeSections.length - 1];
   }
 
+  detectRangeMarkup(range, markupTagName) {
```

Comment:
makes sense. I was confused because elsewhere the variable name for the result of `detectRangeMarkup` was being called `hasMarkup`